HLTrigger/Configuration/tables/subtables.sh

```diff
@@ -110,6 +110,20 @@ function loadConfiguration() {
       DBUSER="cms_hlt_v3_w"
       PWHASH="0196d34dd35b04c0f3597dc89fbbe6e2"
       ;;
+      "v3-test/dev")
+      # v3-test dev
+      DBHOST="cmsr1-s.cern.ch"
+      DBNAME="cms_hlt.cern.ch"
+      DBUSER="cms_hlt_gdrdev_w"
+      PWHASH="0196d34dd35b04c0f3597dc89fbbe6e2"
+      ;;
+      "v3/dev")
+      # v3 dev
+      DBHOST="cmsr1-s.cern.ch"
+      DBNAME="cms_hlt.cern.ch"
+      DBUSER="cms_hlt_gdrdev_w"
+      PWHASH="0196d34dd35b04c0f3597dc89fbbe6e2"
```

Comment:
It seems "v3-test/dev" and "v3/dev" have the very same parameters, shouldn't there be a difference?

Reply:
dev is the database and thats what the parameters reflect.  v3 and v3-test are the converters to use. Honestly should have been rewritten to remove the converter from the string but meh...

Reply:
OK, thanks for the clarification